frontend/src/App.js

```diff
@@ -1,57 +1,56 @@
-import React, {useState} from 'react';
-import './App.css';
-import {BrowserRouter as Router, Route, Switch} from "react-router-dom";
-import Home from './components/Home/Home'
-import MyProjects from './components/Projects/MyProject';
-import WhyPerflow from './components/WhyPerflow/WhyPerflow';
+import React, { useState } from "react";
+import "./App.css";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+import Home from "./components/Home/Home";
+import MyProjects from "./components/Projects/MyProject";
+import WhyPerflow from "./components/WhyPerflow/WhyPerflow";
 
-import ProjPanel from './components/Projects/ProjPanel';
-import EditProject from './components/Projects/EditProject';
+import ProjPanel from "./components/Projects/ProjPanel";
+import EditProject from "./components/Projects/EditProject";
 
-import Navbar from './components/Navbar';
+import Navbar from "./components/Navbar";
```

Comment:
The recommendable is use a single quote. So, it was fine before but it's okay

Reply:
Also you only changed it in one component (App.js) so this creates inconsistency in our code. I suggest to put it back as it was before or someone eventually will do it in a different task.